### PR TITLE
added jax.numpy.round function

### PR DESCRIPTION
**Pull request by x0axz**
Close https://github.com/unifyai/ivy/issues/12370

### CODE REVIEW

**Comment by x0axz** [comment]
@RickSanchezStoic Please review this PR.

**Comment by x0axz** [comment]
@RickSanchezStoic Please review this PR.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by x0axz** [comment]
@RickSanchezStoic Please review this PR.
